gulpfile.js

```diff
@@ -364,3 +364,12 @@ gulp.task('validateTelemetryMD', async () => {
         throw new Error('Telemetry MD is not valid, please re-run `npm run generateTelemetry`');
     }
 });
+
+gulp.task('verifyUnhandledErrors', async () => {
+    const fileName = path.join(__dirname, 'unhandledErrors.txt');
```

Comment:
Optionally, this will ensure we no longer have unhandled exceptions in our new code.